x-pack/plugin/ml-feature-index-builder/src/test/java/org/elasticsearch/xpack/ml/featureindexbuilder/action/StopFeatureIndexBuilderJobActionRequestTests.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.ml.featureindexbuilder.action;
+
+import org.elasticsearch.common.xcontent.XContentParser;
+import org.elasticsearch.test.AbstractXContentTestCase;
+import org.elasticsearch.xpack.ml.featureindexbuilder.action.StopFeatureIndexBuilderJobAction.Request;
+import java.io.IOException;
+
+public class StopFeatureIndexBuilderJobActionRequestTests extends AbstractXContentTestCase<Request> {
```

Comment:
I think it should extend `AbstractStreamableTestCase` instead of `AbstractXContentTestCase`.  As it stands at the moment this test is not testing that the wire streaming read/write methods are consistent.